CODEOWNERS

```diff
@@ -78,3 +78,9 @@ tsconfig.json @brunobar79 @osdnk @estebanmino @nickbytes
 
 # SECURITY / APP INFRA
 yarn.lock @brunobar79 @estebanmino @nickbytes
+
+# APP / NOTIFICATIONS
+src/notifications @jkadamczyk
```

Comment:
no, this is not needed, this is not supersensitive. (Also aside, having one code owner per file is asking for trouble. There will be no way to edit those files when you'll be on vacation)